guides/kontrol-example/compositional-symbolic-execution.md

```diff
@@ -125,114 +112,203 @@ kontrol build
+┃  │   statusCode: EVMC_SUCCESS
+┃  │   src: test/Arithmeric.t.sol:4:6
+┃  │   method: test%Arithmetic.add(uint256,uint256)
+┃  │
+┃  ┊  constraint:
+┃  ┊      ( notBool <acctID>
+  C_ARITHMETIC_ID:Int
+</acctID> in_keys ( ACCOUNTS_REST:AccountCellMap ) )
+┃  ┊  subst: ...
+┃  └─ 2 (leaf, target, terminal)
+┃      k: #halt ~> CONTINUATION:K
+┃      pc: PC_CELL_5d410f2a:Int
+┃      callDepth: CALLDEPTH_CELL_5d410f2a:Int
+┃      statusCode: STATUSCODE_FINAL:StatusCode
+┃
+┗━━┓ subst: .Subst
+   ┃ constraint:
+   ┃     ( maxUInt256 -Int KV1_b:Int ) <Int KV0_a:Int
+   │
+   ├─ 9
+   │   k: #execute ~> CONTINUATION:K
+   │   pc: 0
+   │   callDepth: CALLDEPTH_CELL:Int
+   │   statusCode: STATUSCODE:StatusCode
+   │   src: test/Arithmeric.t.sol:3:20
+   │   method: test%Arithmetic.add(uint256,uint256)
+   │
+   │  (999 steps)
+   ├─ 7 (terminal)
+   │   k: #halt ~> CONTINUATION:K
+   │   pc: 605
+   │   callDepth: CALLDEPTH_CELL:Int
+   │   statusCode: EVMC_REVERT
+   │   method: test%Arithmetic.add(uint256,uint256)
+   │
+   ┊  constraint:
+   ┊      ( notBool <acctID>
+  C_ARITHMETIC_ID:Int
+</acctID> in_keys ( ACCOUNTS_REST:AccountCellMap ) )
+   ┊  subst: ...
+   └─ 2 (leaf, target, terminal)
+       k: #halt ~> CONTINUATION:K
+       pc: PC_CELL_5d410f2a:Int
+       callDepth: CALLDEPTH_CELL_5d410f2a:Int
+       statusCode: STATUSCODE_FINAL:StatusCode
+```
 
-{% hint style="warning" %}
-**Experimental Feature**: NatSpec preconditions are currently experimental. While most popular Solidity terms and variable types are supported, some advanced features may not be available. If you need support for specific Solidity constructs, please reach out to the community or consider contributing to the project.
-{% endhint %}
+As you can see, the summary covers two branches: one where the addition succeeds, and one where it overflows and reverts. During the execution of the test, this summary is being used as a high-priority semantic rule that gets inserted into the rule base (see [K Tutorial](https://kframework.org/docs/user_manual/) for more details on how rules are applied). Whenever the `add` function is called, this summary rule is applied instead of symbolically executing the function's implementation, directly determining the state transformation that corresponds to the function execution. 
 
-Kontrol supports special NatSpec comments to specify preconditions directly in your Solidity code using the `@custom:kontrol-precondition` annotation. This feature allows you to express constraints on function parameters, storage variables, and global variables that are automatically applied during symbolic execution.
+If you look at the KCFG of the `testAdd` function, you will see that it does not contain nodes representing the execution of the `add` function, such as executing the `CALL` instruction, etc., but rather the target node(s) representing the result of this function execution:
+```bash 
+kontrol show 'ArithmeticTest.testAdd('
+```
```

Comment:
Do you know if this is duplicated? As it's the same command from the beginning of the section, and does not have a kcfg below.